build.gradle

```diff
@@ -257,34 +261,63 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
     compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: versions.springHystrix
     compile group: 'org.springframework.retry', name: 'spring-retry', version: '1.2.5.RELEASE'
+    compile('org.springframework.cloud:spring-cloud-starter-openfeign')
```

Comment:
i believe if you are including spring-bom for dependency management then there is no need to include version for spring dependencies